ui/components/src/StatChart/StatChart.tsx

```diff
@@ -113,31 +112,33 @@ export function StatChart(props: StatChartProps) {
   const charactersAdjust = formattedValue.length;
   const valueSize = isLargePanel === true ? MAX_VALUE_SIZE : Math.min(width, height) / charactersAdjust;
 
+  const containerPadding = `${chartsTheme.container.padding.default}px`;
+
   return (
-    <Box>
+    <Stack sx={{ height: '100%', width: '100%' }} spacing={0}>
```

Comment:
It does some flexbox things for us, so the two values sit nicely together to fill the space. I could also set the flex values on `Box` in `sx` if that'd be cleaner. I'm still getting used to working with MUI. Let me know if you think that'd be better.

Reply:
Personally I like setting `flex` on `Box` or just `div` better because `Stack` usually means you want even spacing between elements and i've heard there's performance issue with using too many `Box` and `Stack`. But i don't feel strongly about this 🤷‍♀️

Reply:
That's good to know! I wasn't aware of the perf bits since I'm still learning MUI. Let me take a quick stab at just doing the flex on the `Box`.